low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -165,7 +165,7 @@ NODEHWSENSOR:
 
 REALSTORLOGICALVOLUMESENSOR:
    threaded: true
-   polling_frequency_override: 0
+   polling_frequency_override: 10
```

Comment:
why logical volume, only disk group alerts are  needed faster

Reply:
Yeah. @sandeepsgt suggested to create a new section in config.

Reply:
@cdeshmukh It is REALSTORLOGICALVOLUMESENSOR is disk group only

Reply:
I get it, there is one sensor for both DG & logical volume and same applies for [ polling frequency](https://github.com/Seagate/cortx-monitor/blob/main/low-level/sensors/impl/platforms/realstor/realstor_dg_volume_sensor.py#L137).

Lets correct that to polling frequency config also then, rename to 'pollfreq_DG_logical_volume_sensor' and in config file, keep over ride under DG only, because first DG is polled and then related logical volume with same frequency

Reply:
Done.